autorun.brs

```diff
@@ -291,16 +280,19 @@ Sub EnterEventLoop()
           receivedLoadFinished = true
         else if eventData.reason = "message" then
           DebugLog("BS: Message receved: " + FormatJson(eventData.message, 0))
-          if eventData.message.screenshotInterval <> invalid then
-            gaa.screenshotTimer.SetElapsed((val(eventData.message.screenshotInterval) / 1000), 0)
-            DebugLog("BS: Updated screenshot interval")
-          else if eventData.message.isScreenshotEnabled <> invalid then
-            if eventData.message.isScreenshotEnabled = "true" then
-              gaa.screenshotTimer.Start()
```

Comment:
same here: screenshots were enabled only until the player was rebooted but the toggle would be still ON in the CMS

Reply:
the screenshot is now made on PlayerJS side using BrightSign JS API